pkg/services/rpcsrv/server.go

```diff
@@ -220,7 +220,9 @@ var rpcHandlers = map[string]func(*Server, params.Params) (any, *neorpc.Error){
 	"getpeers":                     (*Server).getPeers,
 	"getproof":                     (*Server).getProof,
 	"getrawmempool":                (*Server).getRawMempool,
+	"getrawnotarypool":             (*Server).getRawNotaryPool,
 	"getrawtransaction":            (*Server).getrawtransaction,
+	"getrawnotaryrequest":          (*Server).getRawNotaryRequest,
```

Comment:
Let's rename it to `getrawnotarytransaction` to match the https://github.com/nspcc-dev/neo-go/issues/2951#issuecomment-1677305675.

Reply:
Fixed.